tests/client/test_treeherder_client.py

```diff
@@ -511,5 +522,43 @@ def test_treeheder_auth(self, mock_time, mock_generate_nonce):
                                                "user=project&"
                                                "oauth_signature=DJe%2F%2FJtw7s2XUrciG%2Bl1tfJJen8%3D"))
 
+    @patch("treeherder.client.client.requests.get")
+    def test_get_job(self, mock_get):
+
+        mock_get.return_value = self._get_mock_response({
+            "meta": {"count": 3,
+                     "repository": "mozilla-inbound",
+                     "offset": 0},
+            "results": [{"jobDetail1": 1},
+                        {"jobDetail2": 2},
+                        {"jobDetail3": 3}
+                        ]})
+        tdc = TreeherderClient()
+        jobs = tdc.get_jobs("mozilla-inbound")
+        self.assertEqual(len(jobs), 3)
+        self.assertEqual(jobs, [{"jobDetail1": 1},
+                                {"jobDetail2": 2},
+                                {"jobDetail3": 3}
+                                ])
+
+    @patch("treeherder.client.client.requests.get")
+    def test_get_results(self, mock_get):
+
+        mock_get.return_value = self._get_mock_response({
+            "meta": {"count": 3, "repository": "mozilla-inbound",
+                     "offset": 0},
+            "results": [{"resultSet1": 1},
+                        {"resultSet2": 2},
+                        {"resultSet3": 3}
+                        ]})
+
+        tdc = TreeherderClient()
+        resultSets = tdc.get_resultsets("mozilla-inbound")
+        self.assertEqual(len(resultSets), 3)
+        self.assertEqual(resultSets, [{"resultSet1": 1},
```

Comment:
Same as above: better to define what you expect to see in one place, instead of repeating it over and over.